edgelet/contrib/debian8/postinst

```diff
@@ -10,9 +10,13 @@ case "$1" in
             chown ${USER}:${GROUP} /var/lib/iotedge
             chown ${USER}:${GROUP} /var/log/iotedge
             chown ${USER}:${GROUP} /etc/iotedge/config.yaml
+            if [ ! -f /etc/iotedge/config.yaml ]; then
+              cp -a /etc/iotedge/config.yaml.template /etc/iotedge/config.yaml
```

Comment:
Indentation is wrong. The rest of the file uses 4 spaces.

Reply:
fixed